roles/splunk_common/tasks/install_splunk.yml

```diff
@@ -10,6 +10,17 @@
   - "{{ manifests.files }}"
   when: splunk_upgrade | bool
 
+- name: Remove old python files
+  file:
+    path: "{{ item.path }}"
+    state: "absent"
+  ignore_errors: yes
+  become: yes
+  become_user: "{{ privileged_user }}"
+  with_items:
+  - "{{ py_files.files }}"
```

Comment:
Can we use the paths here? It would eliminate the need for searching them, and it already ignores when we try to remove paths that don't exist.

Reply:
I think I might do something even more involved tomorrow.  We'll probably use the manifests file to determine what files should be deleted and remove anything that matches the hash in there.